src/Control/Inciter/InputDeck/Grammar.hpp

```diff
@@ -581,14 +590,37 @@ namespace deck {
                                     eq,
                                     param > {};
 
+  //! put in PDE parameter for equation matching keyword
+  template< typename eq, typename keyword, typename p,
+            class kw_type = tk::grm::number >
+  struct parameter :
+         tk::grm::process< use< keyword >,
+                           tk::grm::Store_back< tag::param, eq, p >,
+                           kw_type > {};
+
```

Comment:
I had to move this higher in the file so the new `subsonic_bc` struct below can _see_ it.

Sorry for this confusion here: there _is_ a `parameter` in `tk::grm` (in `Control/CommonGrammar.hpp`), and also another one here.